### PR TITLE
Hire button

### PR DESCRIPTION
**Pull request by himanshu07gupta**
I linked your mail-id with this button.
issue #41

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -68,7 +68,7 @@ <h2>ABOUT ME</h2>
 
 </p>
                 <br>
-                <a href="" class="homeBtn">HIRE ME</a>
+                <a href="https://mail.google.com/mail/u/0/#sent?compose=GTvVlcSHxTXzSsMSLMZPVlDQZLztRrhSVNCjGGpfLdLNklTNSNzSJVnMPCJWgCgpHwgKgXBVDFJWM" class="homeBtn">HIRE ME</a>
```
Comment (Iamdivyak) [inline about.html]: it's not the right way to do it. @himanshu07gupta. 
href="mailto:kumaridivya2796@gmail.com"
remove href with it.

**Review by Iamdivyak — CHANGES_REQUESTED** [review]

File: about.html (hunk 1 above)
Comment (himanshu07gupta) [reply about.html]: ok sure

**Comment by Iamdivyak** [comment]
Thanks for contributing @himanshu07gupta

**Comment by himanshu07gupta** [comment]
> Thanks for contributing @himanshu07gupta

Please add Hacktoberfest label on my pull request.

**Pull request merged**